db_scripts/mosip_master/ddl/master-valid_document.sql

```diff
@@ -17,7 +17,7 @@
 CREATE TABLE master.valid_document(
 	doctyp_code character varying(36) NOT NULL,
 	doccat_code character varying(36) NOT NULL,
-	lang_code character varying(3) NOT NULL,
+	lang_code character varying(3),
```

Comment:
What if the able is already created? You should not altering it this way. You should use alter table to drop the null condition.